cart/application/cartService.go

```diff
@@ -47,12 +47,24 @@ type (
 		RestrictionResult validation.RestrictionResult
 	}
 
+	// AdjustmentResult restriction result enriched with the respective item
+	AdjustmentResult struct {
+		Item              cartDomain.Item
```

Comment:
Maybe prefix this struct with Qty to be more explicit?

QtyAdjustmentResult?

Reply:
Changed to QtyAdjustmentResult